openfisca_uk/parameters/consumption/carbon/production/carbon(goods &services).yaml

```diff
@@ -7,7 +7,7 @@ values:
   metadata:
   unit: tonnes CO2
   reference:
-    - title: BEIS 2019 Updated Energy & Emissions Projections
+      title: BEIS 2019 Updated Energy & Emissions Projections
```

Comment:
I think this dash didn't need to be removed; it signifies this reference is one in a list of references.